compiler/optimizer/PreExistence.hpp

```diff
@@ -106,8 +107,8 @@ class TR_PrexArgInfo
    {
    public:
 
-#ifdef J9_PROJECT_SPECIFIC
    static TR_PrexArgInfo* enhance(TR_PrexArgInfo *dest, TR_PrexArgInfo *source, TR::Compilation *comp);
+#ifdef J9_PROJECT_SPECIFIC
```

Comment:
Since this now became an OMR API, can you provide a Doxygen comment for `enhance` describing the API please?